CODE_OF_CONDUCT.md

```diff
@@ -1,10 +1,10 @@
 <!--
 SPDX-FileCopyrightText: 2014 Coraline Ada Ehmke
-SPDX-FileCopyrightText: 2019-2021 Kattni Rembor for Adafruit Industries
+SPDX-FileCopyrightText: 2019-2021 Kattni Rembor
 
 SPDX-License-Identifier: CC-BY-4.0
 -->
-# Adafruit Community Code of Conduct
+# CircuitPython Community Code of Conduct
```

Comment:
Looks like this may have changed due to a community cookiecutter run. Since this is under the adafruit org, then please switch it back.

Reply:
I did update some portions of this library based on another recent library. I figured some of the community standards may have changed.

Reply:
Would you like me to roll back the entire code of conduct or just update that one line?

Reply:
I appreciate you updating it. That's fine. I'd just like to keep it the Adafruit version.